content/docs/querying/basics.md

```diff
@@ -85,7 +85,7 @@ For example, this selects all `http_requests_total` time series for `staging`,
     http_requests_total{environment=~"staging|testing|development",method!="GET"}
 
 Label matchers that match empty label values also select all time series that do
-not have the specific label set at all. Regex-matches are fully anchored.
+not have the specific label set at all. Regex-matches are not anchored.
```

Comment:
Sorry, did not see all the comments first...

I don't feel strongly, but "partial match" _is_ pretty much the correct technical term in my experience...